README.md

```diff
@@ -70,13 +70,13 @@ Mode 2 can be used to verify the Suyama A residue stored in a verifiable delay f
 ```bash
 cofact -cpr F14.proof 14
 ```
-This performs a Pepin primality test for $F_{14}$, then calculates the Suyama A residue and compares it with the residue stored in the proof file. If no residue mismatch is reported, then the proof file has a valid residue. You may check any proof file with or without specifying factors.
+This performs a Pépin primality test for $F_{14}$, then calculates the Suyama A residue and compares it with the residue stored in the proof file. If no residue mismatch is reported, then the proof file has a valid residue. You may check any proof file with or without specifying factors.
 
 Mode 3 can be used to test a Fermat cofactor using the Suyama A residue stored in the corresponding proof file. For example, to test the $F_{14}$ cofactor using the $F_{14}$ proof file, type:
 ```bash
 cofact -upr F14.proof 14 116928085873074369829035993834596371340386703423373313
 ```
-Mode 3 avoids the lengthy Pepin calculation, allowing a cofactor to be tested in a matter of minutes even for Fermat numbers as large as $F_{29}$. To enable yourself to test the resulting cofactor after the next factor of $F_{14}$ through $F_{29}$ is discovered, download the $F_{14}$ through $F_{29}$ proof files from Catherine's excellent [website](https://64ordle.au/fermat/).
+Mode 3 avoids the lengthy Pépin calculation, allowing a cofactor to be tested in a matter of minutes even for Fermat numbers as large as $F_{29}$. To enable yourself to test the resulting cofactor after the next factor of $F_{14}$ through $F_{29}$ is discovered, download the $F_{14}$ through $F_{29}$ proof files from Catherine's excellent [website](https://64ordle.au/fermat/).
```

Comment:
In line 79 I would probably remove the entire comment starting “To enable …”, since at line 81 the mention of the `run_all` script _does_ actually require proofs in the same directory, in order to run properly for F17 and higher: see my further comment there.

Reply:
I wrote the "To enable..." comment to encourage users, who may or may not be interested in running run_all, to download all the proof files before the next factor of F12 through F29 is found, rather than scrambling to find and download the needed proof file after a new factor is reported. Being prepared may be the difference between being the first person to announce a new cofactor is composite (or PRP !!!) vs not.  Since the two paragraphs are making two different points, I am fine with downloading proof files being mentioned in both.

I did notice the "To enable..." sentence only mentioned F14 to F29, so I changed it to F12 to F29 to match your now-available proofs.

Reply:
While maintaining the set of proofs is providing a useful service, I’ll need to keep an eye on my bandwidth – if there is not a huge onrush of people downloading the large files I think I will likely remain within my limits.

The worst cases are (1) I have to move the largest proofs – probably the F28 and F29 proofs will do – somewhere else like archive.org, and/or
(2) I have to pay Oracle for exceeding my bandwidth limits…